docs/DeveloperGuide.md

```diff
@@ -237,7 +237,7 @@ Additionally, the classes implement the following operations:
 Given below is an example usage scenario and how the grade progress feature behaves at each step.
 
 Step 1. The user launches the application for the first time. The `AddressBook` will be initialized with the sample data and
-each person in the `AddressBook` contains an empty `GradeProgressList` object.
+assume each person in the `AddressBook` contains an empty `GradeProgressList` object.
```

Comment:
think this one should be removed, our sample data has grades now

Reply:
yea thats why i added assume thooo cus the use case assumes that empty.

Reply:
doesnt make sense cos it says sample data alr, u cant js assume its empty if its clearly not empty. and yeah means the use case need to change

Reply:
wait which use case

Reply:
ur only talking abt removing line 240 entirely right?